python/cudf/cudf/core/frame.py

```diff
@@ -1759,8 +1759,8 @@ def sample(
                     np.iinfo(np.int64).max, dtype=np.int64
                 )
             elif isinstance(random_state, np.random.mtrand.RandomState):
-                pos = random_state._bit_generator.state["state"]["pos"]
-                keys = random_state._bit_generator.state["state"]["key"]
+                keys = random_state.get_state()[1]
+                pos = random_state.get_state()[2]
```

Comment:
How many items are in the list? Can we do something like:
```suggestion
                _, keys, pos = random_state.get_state()
```

Reply:
There are five items, and its a tuple. https://numpy.org/doc/stable/reference/random/generated/numpy.random.get_state.html#numpy.random.get_state